crates/rpc/rpc-types/src/eth/engine/payload.rs

```diff
@@ -226,6 +286,21 @@ impl From<SealedBlock> for ExecutionPayloadV2 {
     }
 }
 
+impl TryFrom<ExecutionPayloadV2> for Block {
```

Comment:
we want this as functions instead because
we want standalone rpc types eventually

see #4186